api/src/main/java/org/openmrs/module/adminui/utils/RegexUtil.java

```diff
@@ -0,0 +1,60 @@
+package org.openmrs.module.adminui.utils;
+
+/**
+ * Evaluates a string for a given pattern
+ */
+public class RegexUtil {
+
+	/**
+	 * TODO: Confirm the correct pattern for the following regex.
+	 */
+	public static final String MUST_CONTAIN_DIGIT_REGEX = "(?=.*\\d).+";
+	public static final String MUST_CONTAIN_NONE_DIGIT_REGEX = "(?=.*[a-z])(?=.*[A-Z])(?=.*\\d).+";//"(?!^[0-9]*$)(?!^[a-zA-Z]*$)^([a-zA-Z0-9])";
+	public static final String MUST_CONTAIN_MIXED_CASES_REGEX = "(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])";
+
+	private String toEvaluate;
+	private String regex;
+
+	public RegexUtil(){}
```

Comment:
If you want to create instances of this class, then i wouldn't name it RegexUtil, our convention normally is to add Util to classes with static methods only which is not the case here, i would possibly name it something like RegexEvaluator.
I would also get rid of the default constructor(one that takes no argument)